### ISSUE TITLE
ImageUploaderView not working in MS Edge

### ISSUE THREAD

**Issue by laurenwalker** [NCEAS/metacatui#1195]
The ImageUploaderView doesn't render correctly in MS Edge. An exception is caught and logged to the console:

`TypeError: object doesn't support property or Method ToString`

It looks like this error is happening in the dropzone code, not MetacatUI code, but I didn't dig in long enough to 100% verify where it is happening (because debugging in IE takes forever!)

**Comment by robyngit** [comment]
`ImageUploaderView` seems to work fine in the latest version of Edge (18). In older versions, part of the dropzone code caused an error. With [commit da84dc9](https://github.com/NCEAS/metacatui/commit/da84dc9e7b7e708dddd67363cc0acbd643d36906), I no longer get this error in Edge16 or Edge17. However, the `ImageUploaderView` still don't work correctly in Edge 15. @laurenwalker how old of browsers should we provide support for?

**Comment by laurenwalker** [comment]
Thanks Robyn for fixing the bug. I looked at some of our Analytics and on the Arctic Data Center, we only had 11 users in the last year that used Edge 15 or earlier, so I think it's okay to not fix it for those browsers. 

However, when the ImageUploaderView/dropzone fails to render, we should display a message in the view that explains it doesn't work and in a tooltip, explain that their browser may be out-of-date.

**Comment by laurenwalker** [comment]
Hi @robyngit - I know you have switched gears recently, but I was wondering if you ever got a start on this:

> However, when the ImageUploaderView/dropzone fails to render, we should display a message in the view that explains it doesn't work and in a tooltip, explain that their browser may be out-of-date.

**Comment by dataone-website-test** [comment]
Hey @laurenwalker, thanks for the reminder and sorry for leaving that loose end! I haven't started it yet, but I will ASAP.

**Comment by robyngit** [comment]
I fixed this using the `core-js` component, which is loaded in the `ImageUploaderView.` I intended to pull this component in only for users with older browsers, but encountered problems since it doesn't work when it's loaded *after* dropzone is already loaded (as far as I can tell). Another option would be to have a parent view pull in `core-js` before rendering `ImageUploaderView`, and after doing a browser test. However, this strategy could be problematic if the view is ever used in a new context, where the parent view hasn't required `core-js` yet.
